docs/quickstart.rst

```diff
@@ -60,7 +60,7 @@ possible using SQLAlchemy:
                                 backref=db.backref('users', lazy='dynamic'))
 
     # Setup Flask-Security
-    user_datastore = SQLAlchemyUserDatastore(db, User, Role)
+    user_datastore = SQLAlchemyUserDatastore(db.session, User, Role)
```

Comment:
Thanks for the note, @fuhrysteve. I think you misunderstood a bit, so I've changed the example itself to use plain sqlalchemy instead of flask-sqlalchemy. For someone using flask-sqlalchemy, it should be pretty obvious they'd need to pass db.session instead of a 'hand-constructed' session.